<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <arg name="rate" default="100" />
  <arg name="vocab_filepath" default="/home/raytsou/ws/csgo_bot/src/ORB_SLAM2/Vocabulary/ORBvoc.txt" />
  <arg name="settings_filepath" default="/home/raytsou/ws/csgo_bot/src/ORB_SLAM2/Examples/Monocular/Screen.yaml" />
  <arg name="img_topic" default="/screencap/image" />
  <arg name="pose_topic" default="/pose" />
  <arg name="output_frame" default="odom" />

  <node name="Mono" pkg="ORB_SLAM2" type="Mono" output="screen" clear_params="true">
    <param name="rate" value="$(arg rate)" />
    <param name="vocab_filepath" value="$(arg vocab_filepath)" />
    <param name="settings_filepath" value="$(arg settings_filepath)" />
    <param name="img_topic" value="$(arg img_topic)" />
    <param name="pose_topic" value="$(arg pose_topic)" />
    <param name="output_frame" value="$(arg output_frame)" />
  </node>
</launch>
